packages/celotool/src/cmds/deploy/destroy/odis.ts

```diff
@@ -0,0 +1,16 @@
+import { addContextMiddleware, ContextArgv, switchToContextCluster } from 'src/lib/context-utils'
+import { removeHelmRelease } from 'src/lib/odis'
+import { DestroyArgv } from '../destroy'
+
+export const command = 'odis'
+
+export const describe = 'destroy the odis package'
+
+type ODISDestroyArgv = DestroyArgv & ContextArgv
+
+export const builder = addContextMiddleware
+
+export const handler = async (argv: ODISDestroyArgv) => {
```

Comment:
Not at all related to your PR, but it seems like it'd be helpful to check for required env vars in advance. Something that we can add as an improvement later to all celotool cmds.

Reply:
Ya agreed.  It's not ideal to catch them during runtime, but in it's defense, celotool will provide an informative line of what's the missing env var when it exits.